examples/Training/python/Create_Image.py

```diff
@@ -53,6 +54,23 @@ def planeGen():
     planeGen(), "numpy image", sizeZ, sizeC, sizeT, description=desc,
     dataset=None)
 
+print "Created new image: ", i.getId(), i.getName()
```

Comment:
Since this is training material a nicer output would be good here, something more like
`Created new Image:6  Name:"CFPNEAT01_R3D.dv"`
as per the other script.